types/documentdb/index.d.ts

```diff
@@ -182,70 +181,319 @@ export interface AuthOptions {
 export interface Procedure extends UniqueId {
 
     /** The function representing the stored procedure. */
-    body(...params: any[]): void;
+    body(...params: any[]): void | string;
```

Comment:
I see. So, what exactly is the point of having the two properties `body` and `serverScript` that have the same value? (Not blaming you at all: just asking!)

To me the documentation is saying that I could initialise either property. How do they differ?

```ts
interface UserScriptable extends UniqueId {
    /** The user function. Must set one of body or serverscript */
    body?: UserFunction;

    /** The user function. Must set one of body or serverscript */
    serverScript?: UserFunction;
}
```